src/main/java/pl/touk/sputnik/connector/PatchsetBuilder.java

```diff
@@ -0,0 +1,32 @@
+package pl.touk.sputnik.connector;
+
+import org.apache.commons.lang3.math.NumberUtils;
+import org.jetbrains.annotations.NotNull;
+import pl.touk.sputnik.configuration.CliOption;
+import pl.touk.sputnik.configuration.Configuration;
+import pl.touk.sputnik.configuration.GeneralOption;
+import pl.touk.sputnik.configuration.Provider;
+
+import static org.apache.commons.lang3.Validate.isTrue;
+import static org.apache.commons.lang3.Validate.notBlank;
+import static org.apache.commons.lang3.Validate.notNull;
+
+public class PatchsetBuilder {
```

Comment:
\[Checkstyle\] ERROR: Utility classes should not have a public or default constructor\.